app/views/foreman_ansible/job_templates/run_playbook-ansible_default.erb

```diff
@@ -12,6 +12,14 @@ template_inputs:
   description: The playbook to run against given hosts
   advanced: false
   value_type: plain
+- name: check_mode
```

Comment:
If the name is "Check mode", we can still use it in the template by `input('Check mode')` but it looks nicer in the form